lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Task/TaskRuntime.cs

```diff
@@ -99,7 +99,7 @@ public Thread StartTaskOnNewThread()
                     _currentStatus.SetRunning();
                     Logger.Log(Level.Verbose, "Calling into user's task.");
                     var result = _userTask.Call(null);
-                    Logger.Log(Level.Info, "Task Call Finished");
+                    Logger.Log(Level.Verbose, "Task Call Finished");
```

Comment:
This log is actually very useful, at least I always look at it to determine if a task has completed or not. Even during the debug, we may not want to enable Verbose as it would bring too many logs. This is per task level. May be is still OK to keep it as Info.